Tests/SymbolKitTests/SymbolGraph/File.swift

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
did you mean to name this file `ModuleTests.swift`?

Reply:
Yes indeed, fixed. I miss naming files as they are created...